qa/SidechainTestFramework/scutil.py

```diff
@@ -267,31 +267,36 @@ def generate_cert_signer_secrets(seed, number_of_schnorr_keys):
 """
 
 
-def generate_certificate_proof_info(seed, number_of_signer_keys, threshold, keys_paths, is_csw_enabled):
+def generate_certificate_proof_info(seed, number_of_signer_keys, threshold, keys_paths,
+                                    is_csw_enabled, type_of_circuit_number):
     signer_keys = generate_cert_signer_secrets(seed, number_of_signer_keys)
 
     signer_secrets = []
-    signer_public_keys = []
+    public_signing_keys = []
+    public_master_keys = []
     for i in range(len(signer_keys)):
         keys = signer_keys[i]
         signer_secrets.append(keys.secret)
-        signer_public_keys.append(keys.publicKey)
+        public_signing_keys.append(keys.publicKey)
+        public_master_keys.append(keys.publicKey)
```

Comment:
Master public keys should be generated in the same way of the signer keys, they should not be the same.